lib/docs/src/tag.stories.tsx

```diff
@@ -1,24 +1,42 @@
 import { Meta } from "@storybook/react";
 import { Fragment } from "react";
-import { DivPx, Tag } from "../../core/src";
+import { Button, DivPx, Tag } from "../../core/src";
+import { Utils } from "./utils";
+
+const colors: string[] = [
+	"red",
+	"yellow",
+	"green",
+	"blue",
+	"indigo",
+	"purple",
+	"pink",
+	"gray",
+];
 
 export default {
-	title: "Draft/Tag",
+	title: "Components/Tag",
 	component: Tag,
+	argTypes: {
+		children: Utils.arg("text"),
+		color: Utils.arg(colors),
+	},
 } as Meta;
 
-export const Primary = (): JSX.Element => {
-	const colors: string[] = [
-		"red",
-		"yellow",
-		"green",
-		"blue",
-		"indigo",
-		"purple",
-		"pink",
-		"gray",
-	];
+interface Props {
+	children?: string;
+	color?: string;
+}
+
+export const Primary = (props: Props): JSX.Element => {
+	return (
+		<Tag color={(Tag.colors as any)[props.color ? props.color : "gray"]}>
```

Comment:
@thien-do could you give a proper way to use `Tag.colors` here? I don't think that using `as any` is a good way but I still not know which type is fit in this case.

Reply:
We should not use this in our code, but it's totally ok here, and actually it's expected here, since we Storybook requires us to do so.

Hm actually it would be better if we can properly "type" the "props.color" as something like "key Tag.colors", but I'm not sure how to do that.

For now, the above is ok.